test/src/main/java/org/apache/accumulo/test/MissingWalHeaderCompletesRecoveryIT.java

```diff
@@ -161,7 +159,7 @@ public void testEmptyWalRecoveryCompletes() throws Exception {
       // Reading the table implies that recovery completed successfully (the empty file was ignored)
       // otherwise the tablet will never come online and we won't be able to read it.
       try (Scanner s = client.createScanner(tableName, Authorizations.EMPTY)) {
-        assertEquals(0, Iterables.size(s));
+        assertEquals("", new String());
```

Comment:
I think this code can stay the same also, let me know if the test is failing and I can take a look at it.


```suggestion
        assertEquals(0, Iterables.size(s));
```